assets/css/style.css

```diff
@@ -0,0 +1,140 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+/* Set brand colour as background */
+body {
+  background-color: #d9dcd6;
+}
+
+/* Format agency name to represent logo */
+.header {
+  padding: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  background-color: #2a607c;
+  color: #ffffff;
+}
+
+.header h1 {
+  display: inline-block;
+  font-size: 48px;
+}
+
+.header h1 #seo {
+  color: #d9dcd6;
+}
+
+/* Format nav bar to display in a row and remove bullet points and hyperlink defaults */
```

Comment:
CSS comments don't need to describe what the styling is, they are mainly used to let us easily recognise what section of the HTML is being styled. An appropriate comment here would be something like `/* Header and navigation styles */`

Reply:
Thanks Natasha - for this useful note and for your comments on the grading feedback :).  I've now updated the CSS commentary and noted this for my current homework.